.prettierrc

```
@@ -0,0 +1,6 @@
+{
+	"useTabs": true,
+	"singleQuote": true,
+	"jaxSingleQuote": true,
```

Esta línea tiene un typo

```suggestion
	"jsxSingleQuote": true,
```